test/library/packages/UnitTest/TupleComplex.good

```diff
@@ -0,0 +1 @@
+./UnitTest.chpl:231: error: assert failed - 5.0 + 6.0i and (1, 2, 3) are not of same type
```

Comment:
Actually, now that I think about it, it would probably be good to surround all printed variables with `'` or `"`, so that the eye can visually see when it begins and ends

Reply:
Sure. Will do that.